docs/src/mixins/01-typography.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Canvas } from '@storybook/addon-docs/blocks';
+import './01-typography.scss';
+
+<Meta title="Guidelines/Typography" />
+
+## Typography
+
+You can use <a href="https://github.com/wmde/wikit/blob/master/vue-components/src/styles/mixins/Typography.scss">typography mixins</a> to apply WiKit text styles in context and customize any of their available attributes.
```

Comment:
Maybe, but more technical: "You can include the following mixins in your local CSS to create typography classes"

Reply:
I think "any" is correct, but the whole sentence makes it sound confusing. I'll edit. Thanks, Michael!